crates/nu-command/src/commands/run_external.rs

```diff
@@ -3,21 +3,19 @@ use crate::prelude::*;
 
 use derive_new::new;
 use parking_lot::Mutex;
-use std::path::PathBuf;
 
-use nu_engine::shell::CdArgs;
 use nu_engine::WholeStreamCommand;
 use nu_errors::ShellError;
 use nu_protocol::hir::{Expression, ExternalArgs, ExternalCommand, Literal, SpannedExpression};
 use nu_protocol::{Signature, SyntaxShape};
-use nu_source::Tagged;
 
 #[derive(Deserialize)]
 pub struct RunExternalArgs {}
 
 #[derive(new)]
 pub struct RunExternalCommand {
     /// Whether or not nushell is being used in an interactive context
+    #[allow(dead_code)]
     pub(crate) interactive: bool,
```

Comment:
interactive is not used anymore. I allowed dead code, so that I don't have to remove it and adjust every user (makes for an easier diff).